front/src/common/IntervalsDataViz/data.ts

```diff
@@ -361,8 +364,10 @@ export function update<T>(
   targetLine: LineString,
   linearMetadata: Array<LinearMetadataItem<T>>
 ): Array<LinearMetadataItem<T>> {
+  if (sourceLine.coordinates.length !== 0) {
+    return linearMetadata;
```

Comment:
Why did you do this change ?
Why do you return linearMetadata ? (it's only a suggestion, maybe I lack context)
```suggestion
    return [];
```